models/upload.go

```diff
@@ -229,19 +229,19 @@ func (um *UploadManager) CalculateRefundCost(upload *Upload) (float64, error) {
 	}
 	var (
 		startDate  time.Time
-		removeDate = upload.GarbageCollectDate
+		removeDate = upload.GarbageCollectDate.UTC()
 	)
 	if upload.UpdatedAt == nilTime {
-		startDate = upload.CreatedAt
+		startDate = upload.CreatedAt.UTC()
 	} else {
-		startDate = upload.UpdatedAt
+		startDate = upload.UpdatedAt.UTC()
 	}
-	now := time.Now()
+	now := time.Now().UTC()
```

Comment:
>I'm not sure what you mean by this? The tests use a variety of times, including future dates, time.Now, as well as stuff like time.Now().AddDate(0, 0, -1).

I mean this line here, I think during our current testing now is always close to startDate. we should set this to a few months after, and see if the refund is still as expected.

Reply:
ah okay i see what you mean, yea change that.

Reply:
Good catch there was a bug indeed